application/controllers/welcome.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+
+welcome_fields = {
+    'welcome': fields.String
+}
+
+class WelcomeResources(Resource):
+    def get(self):
+        return marshal('Hello, World', welcome_fields)
```

Comment:
Adding this in as a placeholder so that the root returns something & allows us to deploy. Need to spend some more time on this to return something more appropriate.

Reply:
could be the name of our app and some credits ; )